usaspending_api/recipient/delta_models/sam_recipient.py

```diff
@@ -1,29 +1,116 @@
 SAM_RECIPIENT_COLUMNS = {
-    "awardee_or_recipient_uniqu": "STRING",
+    "id": "LONG NOT NULL",
+    "recipient_hash": "STRING",
     "legal_business_name": "STRING",
-    "ultimate_parent_unique_ide": "STRING",
-    "ultimate_parent_legal_enti": "STRING",
-    "broker_duns_id": "INTEGER NOT NULL",
-    "update_date": "DATE NOT NULL",
+    "duns": "STRING",
     "address_line_1": "STRING",
     "address_line_2": "STRING",
+    "business_types_codes": "ARRAY<STRING>",
     "city": "STRING",
     "congressional_district": "STRING",
     "country_code": "STRING",
+    "parent_duns": "STRING",
+    "parent_legal_business_name": "STRING",
     "state": "STRING",
-    "zip": "STRING",
     "zip4": "STRING",
-    "business_types_codes": "ARRAY<STRING>",
-    "dba_name": "STRING",
-    "entity_structure": "STRING",
+    "zip5": "STRING",
+    "alternate_names": "ARRAY<STRING>",
+    "source": "STRING NOT NULL",
+    "update_date": "TIMESTAMP NOT NULL",
     "uei": "STRING",
-    "ultimate_parent_uei": "STRING",
```

Comment:
Looks like these columns were updated to match the source `sam_recipient` table from broker, however these should match the target delta model. Can you change these back?